README.md

```diff
@@ -287,6 +287,100 @@ However, this will rebuild entire VM, which means that besides your own scripts,
 $ capstan package compose --update node-micro/tutorial
 ```
 
+## Deploying Unikernels on Kubernetes
+You can deploy `osv-microservice-demo` application on Kubernetes at ease. If you don't have a working Kubernetes cluster up-and-running with [Virtlet](https://github.com/Mirantis/virtlet) plugin installed,
+please follow these quick steps.
+
+### Setup local Kubernetes cluster
+
+Install `kubectl` client program:
+```bash
+$ curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+$ cat <<EOF >/etc/apt/sources.list.d/kubernetes.list
+deb http://apt.kubernetes.io/ kubernetes-xenial main
+EOF
+$ apt-get update
+$ apt-get install -y kubectl
+```
+
+Configure and run fake Kubernetes cluster:
+```bash
+$ git clone git@github.com:Mirantis/virtlet.git
+$ cd deploy
+$ ./demo.sh
+```
+
+Run Kubernetes GUI server:
+```bash
+$ kubectl proxy
+# then visit http://localhost:8001/ui to browse Kubernetes UI
+```
+
+Test if everything works:
+```bash
+$ kubectl get pods --namespace kube-system
+NAME                                    READY     STATUS    RESTARTS   AGE
+etcd-kube-master                        1/1       Running   1          4h
+image-server-1782580915-zv2w3           1/1       Running   0          4h
+kube-apiserver-kube-master              1/1       Running   1          4h
+kube-controller-manager-kube-master     1/1       Running   0          4h
+kube-dns-3946503078-zplxw               3/3       Running   0          4h
+kube-proxy-3r4p3                        1/1       Running   0          4h
+kube-proxy-qck08                        1/1       Running   0          4h
+kube-proxy-sw8rm                        1/1       Running   0          4h
+kube-scheduler-kube-master              1/1       Running   1          4h
+kubernetes-dashboard-2396447444-hn0jr   1/1       Running   0          4h
+virtlet-8glgr                           1/1       Running   0          4h
+```
+Notice the `image-server-1782580915-zv2w3` here since we will be uploading OSv images to it.
+
+### Deploy microservices
+
+Now you can deploy the `osv-microservice-demo` application. First you need to compose OSv unikernels
+and upload them to the imageserver. Kubernetes will be able to find them there when it will be
+creating pods. Use utility script `upload_images.sh` for this:
+```bash
+$ npm install
+$ ./virtlet_deploy/upload-images.sh image-server-1782580915-zv2w3
+```
+Deploy microservices to Kubernetes:
+
+```bash
+$ kubectl create -f ./virtlet_deploy/micro-services.sh
```

Comment:
Please add a comment above each of these lines so that users will understand what's the difference between `micro-services` and `micro-deployments` without looking into the scripts.